### PR TITLE
Ran build/versionize-docs.sh

### PR DESCRIPTION
**Pull request by bgrant0607**
Ref #15893

### CODE REVIEW

File: api/swagger-spec/v1beta1.json [hunk 1]
```diff
@@ -2682,7 +2682,7 @@
...
-      "description": "Host is the fully qualified domain name of a network host, as defined by RFC 3986. Note the following deviations from the \"host\" part of the URI as defined in the RFC: 1. IPs are not allowed. Currently an IngressRuleValue can only apply to the\n\t  IP in the Spec of the parent Ingress.\n2. The `:` delimiter is not respected because ports are not allowed.\n\t  Currently the port of an Ingress is implicitly :80 for http and\n\t  :443 for https.\nBoth these may change in the future. Incoming requests are matched against the Host before the IngressRuleValue."
+      "description": "Host is the fully qualified domain name of a network host, as defined by RFC 3986. Note the following deviations from the \"host\" part of the URI as defined in the RFC: 1. IPs are not allowed. Currently an IngressRuleValue can only apply to the\n\u0009  IP in the Spec of the parent Ingress.\n2. The `:` delimiter is not respected because ports are not allowed.\n\u0009  Currently the port of an Ingress is implicitly :80 for http and\n\u0009  :443 for https.\nBoth these may change in the future. Incoming requests are matched against the Host before the IngressRuleValue."
```
Comment (nikhiljindal) [inline api/swagger-spec/v1beta1.json]: Versionize-docs should not have changed this?

**Comment by nikhiljindal** [comment]
Travis is also complanining about the swagger-spec change. Was it auto-generated?

File: api/swagger-spec/v1beta1.json (hunk 1 above)
Comment (bgrant0607) [inline api/swagger-spec/v1beta1.json]: That's unicode for tab:
http://www.fileformat.info/info/unicode/char/0009/index.htm

Our description strings shouldn't contain newlines and tabs. Let's fix that in head.

**Comment by bgrant0607** [comment]
Yes, the swagger change was autogenerated. Reverted.

**Comment by nikhiljindal** [comment]
LGTM, thanks!
Travis should pass now

**Comment by bgrant0607** [comment]
Travis passed

**Comment by bgrant0607** [comment]
Merging

**Pull request merged**